libportal/notification.c

```diff
@@ -195,10 +598,15 @@ get_properties_cb (GObject      *source_object,
 
   while (g_variant_iter_loop (&iter, "{&sv}", &key, &value))
     {
-      if (strcmp (key, "version"))
+      if (strcmp (key, "version") == 0)
         portal->notification_interface_version = g_variant_get_uint32 (value);
+      else if (strcmp (key, "SupportedOptions") == 0)
+        portal->supported_notification_options = g_steal_pointer (&value);
```

Comment:
This will leak a previously set `portal->supported_notification_options`.

Reply:
true, but this function is only called once. Expect if xdp doesn't have a version property. So maybe still good to free the previous value.